src/Components/LogEditForm.js

```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from "react";
+import { useParams, Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+
+function LogEditForm() {
+  let { index } = useParams();
+
+  const [log, setLog] = useState({
+    Captainname: "",
+    title: "",
+    Post: "",
+    daysSinceLastCrisis: "",
+    mistakesWereMadeToday: false,
+  });
+  let navigate = useNavigate();
+  const handleTextChange = (event) => {
+    setLog({ ...log, [event.target.id]: event.target.value });
+  };
+
+  const handleCheckboxChange = () => {
+    setLog({ ...log, mistakesWereMadeToday: !log.mistakesWereMadeToday });
+  };
+
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3333/logs/${index}`)
+      .then((res) => {
+        setLog(res.data);
+        console.log(res.data)
+      })
+      .catch((e) => {
+        console.log(e);
+      });
+  }, [index]);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    axios
+      .put(`http://localhost:3333/logs/${index}`, log)
+      .then((res) => {
+        setLog(res.data);
+        navigate(`/logs/${index}`);
+      })
+      .catch((e) => {
+        console.log(e);
+      });
+  };
+
+  return (
+    <div className="Edit">
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="captainName">Captain's Name:</label>
+        <input
+          id="captainName"
+          value={log.captainName}
```

Comment:
`captainName` does not match what you have in the state object, which is `Captainname` It is causing the app to throw some warnings at you